src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
```

Comment:
Don't use static methods in your solution. It's better to create an instance of a class which method you want to call. Consider refactoring this to an instance method and using an instance of the `Lottery` class to call `getRandomBall`.